DOCS-3196: Update Camera interface for Go Image function and remove Stream/Next (#3854)
viamrobotics/docs: static/include/components/apis/overrides/methods/go.camera.Image.after.md

> You can also directly decode as an `Image.Image` with the camera's `DecodeImageFromCamera` function:

Reviewer: I just looked at the linked [Go SDK doc](https://pkg.go.dev/go.viam.com/rdk/components/camera#VideoSource) to see if there was more about why someone would do one versus the other and noticed it says "Or try to directly decode as an image.Image:" The word "try" makes me wonder if there's a reason the DecodeImageFromCamera method would be more likely to fail than the other method. Is there?

Author: @randhid is there any reason we should update the wording here to account for the method failing?

> You can also directly decode as an `Image.Image` with the camera's `DecodeImageFromCamera` function:

Author: ```suggestion
You can also try to directly decode as an `Image.Image` with the camera's `DecodeImageFromCamera` function:
```

```go {class="line-numbers linkable-line-numbers"}
myCamera, err := camera.FromRobot(machine, "my_camera")
img, err = camera.DecodeImageFromCamera(context.Background(), utils.MimeTypeJPEG, nil, myCamera)
```

> If you use this method, be sure to import `"go.viam.com/rdk/utils"` at the beginning of your file.

Reviewer: I think?
```suggestion
To use either method, be sure to import `"go.viam.com/rdk/utils"` at the beginning of your file.
```

Author: good catch, thanks!
